games/l.yaml

```diff
@@ -120,7 +120,7 @@
 
 - name: [Locomotion (Amiga 1992), 'http://www.mobygames.com/game/locomotion']
   meta:
-    genre: [simulation]
+    genre: [Strategy, Puzzle]
```

Comment:
I've mentioned before why having a bland "Strategy" genre is not a good idea. A quick search has showed this game to be arcade-puzzle, which I think it's appropriate. See description here https://www.youtube.com/watch?v=im2CoOo4D3o